CONTRIBUTING.md

```diff
@@ -63,8 +63,6 @@ For each release, the following needs to be done:
 * Steps to perform *before* pushing a release tag up:
```

Comment:
I would unwind this to just two parental bullet points tbh. It made sense when it had four sub-items, but now it looks unnecessary imho.

Reply:
Resolving this one on your behalf @maratal.